### ISSUE TITLE
Deep learning for healthcare: review, opportunities and challenges

### ISSUE THREAD

**Issue by alxndrkalinin** [greenelab/deep-review#423]
https://doi.org/10.1093/bib/bbx044

> Gaining knowledge and actionable insights from complex, high-dimensional and heterogeneous biomedical data remains a key challenge in transforming health care. Various types of data have been emerging in modern biomedical research, including electronic health records, imaging, -omics, sensor data and text, which are complex, heterogeneous, poorly annotated and generally unstructured. Traditional data mining and statistical learning approaches typically need to first perform feature engineering to obtain effective and more robust features from those data, and then build prediction or clustering models on top of them. There are lots of challenges on both steps in a scenario of complicated data and lacking of sufficient domain knowledge. The latest advances in deep learning technologies provide new effective paradigms to obtain end-to-end learning models from complex data. In this article, we review the recent literature on applying deep learning technologies to advance the health care domain. Based on the analyzed work, we suggest that deep learning approaches could be the vehicle for translating big biomedical data into improved human health. However, we also note limitations and needs for improved methods development and applications, especially in terms of ease-of-understanding for domain experts and citizen scientists. We discuss such challenges and suggest developing holistic and meaningful interpretable architectures to bridge deep learning models and human interpretability.

I only scrolled through this, but looks like fairly brief review touching upon main DL architectures, applications in imaging, EHR, genomics, mobile health, and challenges (similar to ours).

**Comment by agitter** [comment]
I only skimmed it myself.  At a glance, it looks good, but their scope is much more limited than ours so it serves a different purpose.

They focus on 32 primary papers spanning:
- Clinical imaging
- EHRs
- Genomics
- Mobile

We don't cover mobile (e.g. wearable sensors).  Referring to this review to note it as a relevant research area would be nice.

They also discuss some of the same challenges and opportunities as us, but we have explored these in greater depth.  For instance, they have two paragraphs on interpretability.  Our #368 is essentially its own mini-review on the topic.
